site/src/components/SignInForm/SignInForm.tsx

```diff
@@ -101,18 +108,39 @@ export const SignInForm: FC<SignInFormProps> = ({
     onSubmit,
     initialTouched,
   })
-  const getFieldHelpers = getFormHelpersWithError<BuiltInAuthFormValues>(form, authError)
+  const getFieldHelpers = getFormHelpersWithError<BuiltInAuthFormValues>(
+    form,
+    loginErrors.authError,
+  )
 
   return (
     <>
       <Welcome />
       <form onSubmit={form.handleSubmit}>
         <Stack>
-          {authError && (
-            <ErrorSummary error={authError} defaultMessage={Language.authErrorMessage} />
+          {loginErrors.authError && (
+            <ErrorSummary
+              error={loginErrors.authError}
+              defaultMessage={Language.authErrorMessage}
+            />
+          )}
+          {loginErrors.getUserError && (
+            <ErrorSummary
+              error={loginErrors.getUserError}
+              defaultMessage={Language.getUserErrorMessage}
+            />
+          )}
+          {loginErrors.checkPermissionsError && (
+            <ErrorSummary
+              error={loginErrors.checkPermissionsError}
+              defaultMessage={Language.checkPermissionsErrorMessage}
+            />
```

Comment:
I would DRY this up by mapping over the keys of `loginErrors`

Reply:
Thought about it, but we’ll have to map default messages in a separate object for that. I’ll try that!

Reply:
I was thinking if you map over the keys you could do `defaultMessage={Language[errorKey]` if you name them the same

Reply:
Declared an `enum` with error types, and used a `Record<enum, Error>` for iterating through keys. I was not able iterate through the properties of a `type` or `interface`.